cypress/e2e/shared/explorer.test.ts

```diff
@@ -1035,7 +1037,10 @@ describe('DataExplorer', () => {
 
   describe('refresh', () => {
     beforeEach(() => {
-      cy.writeData(lines(10))
+      cy.get('@org').then(({name: orgName}: Organization) => {
+        cy.writeData(lines(10), orgName, 'defbuck')
```

Comment:
if we can provision `defbuck` why can't we provision the org name?

Reply:
provision?

Reply:
ie: allocate a resource within the system with an initial value at some stage in the cluster's lifecycle

Reply:
the orgname's are different for each user but bucketName is same.